tests/mock_tables/asic2/ip_special_recursive_route.json

```diff
@@ -0,0 +1,35 @@
+{
+    "193.11.208.0/25": [
```

Comment:
Confirming if this is the exact json output to be tested .. I see the following in the switch 
```
    "asic4": {
        "193.11.208.0/25": [
            {
                "destSelected": true, 
                "distance": 200, 
                "installed": true, 
                "internalFlags": 13, 
                "internalNextHopActiveNum": 2, 
                "internalNextHopNum": 2, 
                "internalStatus": 16, 
                "metric": 0, 
                "nexthops": [
                    {
                        "active": true, 
                        "afi": "ipv4", 
                        "fib": true, 
                        "flags": 3, 
                        "interfaceIndex": 128, 
                        "interfaceName": "PortChannel4014", 
                        "ip": "10.1.0.5"
                    }, 
                    {
                        "active": true, 
                        "afi": "ipv4", 
                        "flags": 5, 
                        "ip": "10.0.0.9", 
                        "recursive": true
                    }, 
                    {
                        "active": true, 
                        "afi": "ipv4", 
                        "fib": true, 
                        "flags": 3, 
                        "interfaceIndex": 126, 
                        "interfaceName": "PortChannel4004", 
                        "ip": "10.1.0.1"
                    }, 
                    {
                        "active": true, 
                        "afi": "ipv4", 
                        "flags": 5, 
                        "ip": "10.0.0.5", 
                        "recursive": true
                    }
                ], 
                "prefix": "193.11.208.0/25", 
                "protocol": "bgp", 
                "selected": true, 
                "table": 254, 
                "uptime": "20:17:37"
            }
        ]
    }, 
```

Reply:
Yes.  I am not making it based on the DUT route but fabricating it new one.